decentraland/components/presentational/__snapshots__/BuyLand.test.js.snap

```diff
@@ -41,3 +41,261 @@ exports[`BuyLand renders the component 1`] = `
   </Component>
 </Component>
 `;
+
+exports[`BuyLand renders the component initial state - before start account setup 1`] = `
+<Component
+  style={
+    Object {
+      "alignItems": "center",
+      "backgroundColor": "white",
+      "flex": 1,
+      "justifyContent": "center",
+    }
+  }
+>
+  <LandForSale
+    landForSale={
+      Object {
+        "asset": Object {
+          "id": "0123456789",
+          "img": "https://api.decentraland.org/v1/parcels/0/0/map.png",
+          "name": "Sample Parcel",
+          "type": "PARCEL",
+        },
+        "id": "2",
+        "priceMana": 0,
+        "priceUSD": 0,
+      }
+    }
+  />
+  <Component
+    style={
+      Object {
+        "flexDirection": "row",
+        "marginTop": 66.7,
+      }
+    }
+  >
+    <Button
+      onPress={[Function]}
+      title="Buy"
+    />
+  </Component>
+</Component>
+`;
+
+exports[`BuyLand renders the component waiting for setup - waiting for ETH funding 1`] = `
+<Component
+  style={
+    Object {
+      "alignItems": "center",
+      "backgroundColor": "white",
+      "flex": 1,
+      "justifyContent": "center",
+    }
+  }
+>
+  <LandForSale
+    landForSale={
+      Object {
+        "asset": Object {
+          "id": "0123456789",
+          "img": "https://api.decentraland.org/v1/parcels/0/0/map.png",
+          "name": "Sample Parcel",
+          "type": "PARCEL",
+        },
+        "id": "2",
+        "priceMana": 0,
+        "priceUSD": 0,
+      }
+    }
+  />
+  <Component
+    style={
+      Object {
+        "flexDirection": "row",
+        "marginTop": 66.7,
+      }
+    }
+  >
+    <Button
+      disabled={true}
+      onPress={[Function]}
+      title="Buy"
+    />
+  </Component>
+  <Component
+    style={
+      Object {
+        "flexDirection": "row",
+      }
+    }
+  >
+    <Component>
+      Generating account...
```

Comment:
If the name of this test is waiting for ETH funding, wouldn't that imply that the account has already been generated and we're on the next step of waiting for ETH funding?

Reply:
I think maybe we need a bool for setup in progress and now account generation is complete

Reply:
The test case was buggy. Fixed.

Reply:
Thanks! Resolving this